codespell_lib/data/dictionary.txt

```diff
@@ -18258,6 +18258,7 @@ meerkrat->meerkat
 meetign->meeting
 meganism->mechanism
 mege->merge
+mehod->method
 mehods->methods
 mehtod->method
```

Comment:
You could also add the methodical variant of these if you wanted.

Reply:
Added variants 8bfad59